polkadot/runtime/westend/src/lib.rs

```diff
@@ -1523,9 +1558,26 @@ pub type Migrations = migrations::Unreleased;
 pub mod migrations {
 	use super::*;
 
+	parameter_types! {
+		pub const ImOnlinePalletName: &'static str = "ImOnline";
+	}
+
+	/// Upgrade Session keys to include BEEFY key.
+	/// When this is removed, should also remove `OldSessionKeys`.
+	pub struct UpgradeSessionKeys;
+	impl frame_support::traits::OnRuntimeUpgrade for UpgradeSessionKeys {
+		fn on_runtime_upgrade() -> Weight {
+			if System::last_runtime_upgrade_spec_version() > 103000 {
```

Comment:
where do we skip it? should there be an early return?

Reply:
also this assumes the change will go into the next runtime upgrade, we will need to confirm and sync this with the release/devops team

Reply:
It would be cool to have a `#[run_only_once]` or something like this to be future-proof.

Reply:
Oh that's a really nice catch, thanks!